src/translations/en.json

```
@@ -429,7 +429,7 @@
         "control": "Control",
         "related": "Related",
         "dismiss": "Dismiss",
-        "no_unique_id": "This entity does not have a unique ID, therefore its settings cannot be managed from the UI.",
+        "no_unique_id": "This entity does not have a unique ID, therefore its settings cannot be managed from the UI. This is usually because the entity is defined in configuration.yaml, and settings for those entities must also be done in yaml.",
```

```suggestion
        "no_unique_id": "This entity does not have a unique ID, therefore its settings cannot be managed from the UI. This can be because the entity is defined in configuration.yaml, and setting changes need to be done in that file",
```

I wouldn't change it to this just yet. Just  my suggestion

---

Not sure if this makes it a lot clearer, as some things from `configuration.yaml` are changeable in the UI. And maybe even integrations with config flow will have entities without unique id (is less likely).

---

@bramkragten I don't see the wording conflicting with that.  And if the item is configurable then this error message won't pop up, right?  The problem is when it does pop up, people are left scratching their heads about its meaning.  Certainly open to other wording.  My other idea is that it could be replaced with a link to documentation which explains it.

---

I would vote for a link to a `more-info` page in the documentation.

---

Looks like someone actually created a FAQ over this issue last month, so replaced my text with a link.  I assume linking is allowed within the translation string (I don't have an easy way to verify)?

---

We should not have the link in the translation string.

---

Make it a variable `{doc_link}`

---

And then use the documentation URL function we have in the code base to supply it.

`localize("UI.whaterver.translation", "doc_link", documentationLink("/faq/link_to_doc"))`

---

Also make it an anchor `<a href="...">see the docs</a>

---

I've updated the change, following examples of other doc links